### DISCUSSION TITLE
F1345D0D-32F2-A5DB-24F1-EE28E33832F0

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#151] [General]
# F1345D0D-32F2-A5DB-24F1-EE28E33832F0

Comet Hartley 2 is a Jupiter-family comet orbiting the Sun once every 6.46 years. At 2.25 km in diameter, it is currently the smallest comet ever visited when the Deep Impact spacecraft flew past it in 2010 after its initial target, Comet Boethin, is declared lost.

https://celestia.mobi/addon?item=F1345D0D-32F2-A5DB-24F1-EE28E33832F0

**Comment by Cookie-SpaceGaming118** [comment]
Nice comet
